core/proto/keytransparency_v1_types/keytransparency_v1_types.proto

```diff
@@ -236,3 +236,19 @@ message GetMutationsResponse {
   // results.
   string next_page_token = 7;
 }
+
+// GetMonitoringRequest contains the input parameters of the GetMonitoring APIs.
+message GetMonitoringRequest {
+  // start specifies the start epoch number from which monitoring results will
+  // be returned (ranging from [start, latestObserved] and starting at 1).
+  int64 start = 1;
+}
+
+message GetMonitoringResponse {
+  // smr contains the map root for the sparse Merkle Tree signed with the
```

Comment:
@gdbelvin I've added documenting this in a markdown document to the issue's TODO list so we won't forget about it.

Note to myself: the new inclusion proof (e+1) can be computed using the mutations and the old inclusion proof.